pkg/graph/schema.graphql

```diff
@@ -1317,6 +1321,18 @@ input updateSystemIntakeGRBReviewTypeInput {
   grbReviewType: SystemIntakeGRBReviewType!
 }
 
+"""
+Input data used to set or update a System Intake's GRB Review form data
+"""
+input updateSystemIntakeGRBReviewFormInput {
```

Comment:
Hmm, I think we might want to use the omittable directive on these fields. That allows you to explicitly set the fields to null, (for an example look at the recent work for presentation links) eg `String @goField(omittable: true)`